python/appledoublekaitai/plugin.py

```diff
@@ -4,30 +4,36 @@
 from logbook import Logger
 
 import kaitai_utils
-from structs.apple_single_double import AppleSingleDouble
-
 
 log = Logger(__name__)
 
+BYTE_ARRAY_LENGTH = 256
+
 
 class Plugin(ExtractionPlugin):
 
     def plugin_info(self):
+        file_format = kaitai_utils.get_plugin_title_from_metadata()
+        no_space_plugin_name = file_format.replace(' ', '_')
+        plugin_name = ''.join(letter for letter in no_space_plugin_name if letter.isalnum() or letter == '_')
```

Comment:
These three lines change the plugin name to `applesingle__appledouble`, instead of `'apple_double_kaitai_plugin` (see below).

Do we actually want this to change?

Reply:
Decided to change this back to the original plugin name.